test/common/config/subscription_factory_impl_test.cc

```diff
@@ -230,8 +230,6 @@ TEST_F(SubscriptionFactoryTest, FilesystemCollectionSubscriptionNonExistentFile)
 TEST_F(SubscriptionFactoryTest, LegacySubscription) {
   envoy::config::core::v3::ConfigSource config;
   auto* api_config_source = config.mutable_api_config_source();
-  api_config_source->set_api_type(
```

Comment:
Shouldn't the entire test just be deleted?

Reply:
Yeah.. done. Thnaks for review